app/src/main/java/com/example/mediaplayer/activity/MainActivity.java

```diff
@@ -0,0 +1,225 @@
+     */
+    private void initButtons() {
+        mBtnShuffle = findViewById(R.id.image_button_shuffle);
+        mBtnPrev = findViewById(R.id.image_button_prev);
+        mBtnPlay = findViewById(R.id.image_button_play);
+        mBtnNext = findViewById(R.id.image_button_next);
+        mBtnReplay = findViewById(R.id.image_button_replay);
+        mBtnShuffle.setOnClickListener(this);
+        mBtnPrev.setOnClickListener(this);
+        mBtnPlay.setOnClickListener(this);
+        mBtnNext.setOnClickListener(this);
+        mBtnReplay.setOnClickListener(this);
+    }
+
+    private void initSeekBar() {
+        mSeekBar = findViewById(R.id.seek_bar);
+        mSeekBar.setMax(mMediaController.getmMediaPlayer().getDuration());
+        mSeekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
+            @Override
+            public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
+                if (fromUser) {
+                    mMediaController.jumpTo(progress);
+                    seekBar.setProgress(progress);
+                }
+            }
+            @Override
+            public void onStartTrackingTouch(SeekBar seekBar) { }
+            @Override
+            public void onStopTrackingTouch(SeekBar seekBar) { }
+        });
+        // Thread (Update positionBar & timeLabel)
+        new Thread(new Runnable() {
+            @Override
+            public void run() {
+                while (mMediaController.getmMediaPlayer() != null) {
+                    try {
+                        int currentPosition = mMediaController.getmMediaPlayer().getCurrentPosition();
+                        mSeekBar.setProgress(currentPosition);
+                        runOnUiThread(new Runnable() { // !careful
+                            @Override
+                            public void run() {
+                                int currentPosition = mMediaController.getmMediaPlayer().getCurrentPosition();
+                                mCurrentTime.setText(createTimeLabel(currentPosition));
+                            }
+                        });
+                        Thread.sleep(1000);
+                    } catch (InterruptedException e) {
+                        Log.d("Error", "msg: " + e.getMessage());
+                    }
+                }
+            }
+        }).start();
+    }
+
+    private void initTxtView() {
+        mMaxTime = findViewById(R.id.text_max_time);
+        mCurrentTime = findViewById(R.id.text_current_time);
+        mMaxTime.setText(createTimeLabel(mMediaController.getmMediaPlayer().getDuration()));
+        mCurrentTime.setText(createTimeLabel(0));
+    }
+    /**
+     * @param time (current position of seek bar )
+     * @return String(timeLabel)
+     * @effects: convert position to String format
+     */
+    private String createTimeLabel(int time) {
+        String timeLabel;
+        int min = time / 1000 / 60;
```

Comment:
rm hard code nhé. các số như này nên tạo Hằng số để sử dụng nhé
```suggestion
        int min = time /MILI_SECOND / MINUTE;
```